src/main/webapp/javascript/user-auth.js

```diff
@@ -0,0 +1,96 @@
+var auth2;
+
+function getId() {
+  return auth2.currentUser.get().getId();
+}
+
+/**
+ * loads the auth2 library for the g-signin API, and then initializes
+ * the googleAuth object.
+ * passed as jquery parameter, executed by API script.
+ */
+function onStart() {
+  console.log("onstart");
```

Comment:
Are these `console.log` statements just for your debugging? If so, you can remove them when merging into master.